src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+function App() {
+const todoList = [
+  { id: 1, title: "study" },
+  { id: 2, title: "code" },
+  { id: 3, title: "complete assigment" },
+];
+  return (
+    <div className="App">
+   <h1> Todo List</h1>
+   <ul> 
+       {todoList.map((todo, index) => {
+        return <li key={index}>{todo.title}</li>
+       })}
+
+   </ul>
+   </div>
```

Comment:
Code performs as expected, however, the indentation is a bit inconsistent. neatly formatted code allows a person to read the flow of the code better. . Also, I am wondering why you included the `className="App"`.